docs/package.json

```diff
@@ -15,11 +15,11 @@
     "dotenv": "^6.2.0",
     "exports-loader": "^1.1.1",
     "imports-loader": "^1.2.0",
-    "prettier": "2.3.2",
+    "prettier": "~2.4.1",
```

Comment:
The official recommendation is to use a pinned version of prettier. This is probably fine, but I'd rather err on the side of being cautious because when this causes a problem, it's a weird and hard to identify problem (e.g., the CI saying something that cannot be reproduced locally).